### DISCUSSION TITLE
does this support the devices operating in europe on 868mhz, itron equascan?

### DISCUSSION THREAD

**Discussion by spiri439** [bemasher/rtlamr#198] [Q&A]
I have an itron equan but don't see, to get any data,

**Comment by vogler** [comment]
Just got my old water counters replaced with this: https://www.itron.com/de/solutions/product-catalog/equascan-wmiu-rf
Quick github search for EquaScan did not yield much.
Does it make sense to order some RTL SDR on AliExpress - if so which one is recommended?

**Reply by tpacri** [reply]
Were you able to read it?

**Reply by thanosdr** [reply]
I can read it in rtl_433, but the data is not very honest. RtlAmr, is simply returning nothing, no matter what I try. Its driving me nuts
